srk/test/test_quantifier.ml

```diff
@@ -91,6 +91,130 @@ let strategy2 () =
     assert_bool "strategy2" (check_strategy srk qf_prefix phi strategy)
   | _ -> assert false
 
+(* Exact output formula may differ if miniscope procedure is changed *)
```

Comment:
Could use "assert_equiv_quantified_formula" (plus a simple checker to confirm that existentials do not appear over disjunctions / universals do not appear over conjunctions) to avoid this problem.